.docs/user-guide/storage-providers.md

```diff
@@ -217,19 +247,20 @@ Below is a working `config.yml` file that works with AWS EFS.
 
 ```yaml
 libstorage:
+  service: efs
   server:
     services:
       efs:
         driver: efs
-        efs:
-          accessKey:      XXXXXXXXXX
-          secretKey:      XXXXXXXXXX
-          securityGroups:
-          - sg-XXXXXXX
-          - sg-XXXXXX0
-          - sg-XXXXXX1
-          region:         us-east-1
-          tag:            test
+efs:
```

Comment:
Eh, nevermind this. I thought this was a DigitalOcean PR when I wrote this comment. Stupid me.